EIP-010-wasi-command-runner.md

```diff
@@ -0,0 +1,68 @@
+#  WASI Command Runner
```

Comment:
I think it might make sense to add some kind of general "front matter" to these documents.. 

e.g. to indicate if this EIP is:
```
status: {in-review,accepted,rejected}
implementation: {none,partial,complete}
...
```

Reply:
@bhelx @zshipko what do you think?

Reply:
yeah, i think some additional metadata could be helpful

Reply:
Added in https://github.com/extism/proposals/pull/18/commits/647383c09965cd75b59036022e34a2eeceb9d71c